src/main_functions_home.c

```diff
@@ -1,110 +1,151 @@
-#include <sys/stat.h>
+#include <stdlib.h>
 #include <stdio.h>
-#include <stdlib.h>	
+#include <stdbool.h>
 #include <sys/types.h>
+#include <sys/stat.h>
 #include <string.h>
-#include <stdbool.h>
 #include <assert.h>
 #include "shellstrings.h"
 #include "parser.h"
 #include "main_functions_home.h"
+#include "log.c/src/log.h"
 
 /* See main_functions_home.h */
-void help_page()
-{
-    shell_help();
-	shell_prompt();
+void help_page(bool *color) {
+    shell_help(color);
+	shell_prompt(color);
 
-	parse_read_line();      // accept any input in the command line
+	// Accept any input in the command line
+	parse_read_line();
+	log_trace("exiting the help page.");
 }
 
 /* See main_functions_home.h */
-bool fileexists(const char* filename)
-{
+bool fileexists(const char *filename) {
```

Comment:
Indentation is off in this function. Please run everything through `astyle`.

Reply:
indentation fixed.